src/components/User/UserInfo.jsx

```diff
@@ -0,0 +1,60 @@
+import { Genders, Grades, Majors } from "@/data/Register";
+import { Schools } from "@/data/Schools";
+import Input from "../Input";
+import Radio from "../Radio";
+import Select from "../Select";
+import Button from "../Forms/Button";
```

Comment:
This button should then be stored in the common folder instead of forms, since its being used here as well